src/test/java/org/springframework/data/couchbase/repository/CouchbaseRepositoryQueryIntegrationTests.java

```diff
@@ -68,14 +68,20 @@ void beforeEach() {
 
 	@Test
 	void shouldSaveAndFindAll() {
-		Airport vie = new Airport("airports::vie", "vie", "loww");
-		airportRepository.save(vie);
+		Airport vie = null;
+		try {
+			vie = new Airport("airports::vie", "vie", "loww");
+
+			airportRepository.save(vie);
 
-		List<Airport> all = StreamSupport.stream(airportRepository.findAll().spliterator(), false)
-				.collect(Collectors.toList());
+			List<Airport> all = StreamSupport.stream(airportRepository.findAll().spliterator(), false)
```

Comment:
NIT: Just a suggestion, in case this looks simpler to you:

    List<Airport> all = new ArrayList<>();
    airportRepository.findAll().forEach(target::add);

Or if you find yourself doing this a lot:

    public static <T> List<T> toList(Iterable<T> iterable) {
      List<T> result = new ArrayList<>();
      iterable.forEach(result::add);
      return result;
    }

Reply:
nice.  Fixed.